### DISCUSSION TITLE
Code Table Request - indicate the IPT mapping to DwC term

### DISCUSSION THREAD

**Discussion by mkoo** [ArctosDB/arctos#5280] [Ideas]
**Goal**
Add transparency to how Arctos data is mapped in IPT archives.

Suggested at the Issue Meeting 2/3/2022:  add a new field to Code Tables so we can indicate whether and how a field is mapped to IPT and which DwC term it is addressing. Since Arctos tends to modify some fields for the IPT we need some flexibility to describe this (e.g., I dont thnk just having a link to a DwC term will be sufficient)

Would let us make adjustments like this: #4299 

Would be instructive for new collections and users

**Table**
Pretty much all of them

**Collection type**
all the non-EH, Art, Arch and others which have an DwC archive

**Comment by dustymc** [comment]
This doesn't make sense; code table values are never directly used in DWC, but they may control things that are used in various places in DWC.

I could add comments to https://github.com/ArctosDB/PG_DDL/tree/master/shared_data ??
